spec/foodzone/venues/finder_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../../../app/foodzone/venues/finder'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.